PythonAPI/carla/source/carla/libcarla.pyi

```diff
@@ -0,0 +1,5403 @@
+
+        Args:
+            `intensity (float, optional)`: Intensity of a light (lumens). Defaults to 0.0.\n
+            `color (_type_, optional)`: Color of a light. Defaults to Color.\n
+            `group (_type_, optional)`: Group a light belongs to. Defaults to LightGroup.NONE.\n
+            `active (bool, optional)`: Switch of a light. It is True when the light is on. Defaults to False.\n
+        """
+    # endregion
+
+
+class Location(Vector3D):
+    """Represents a spot in the world."""
+    # region Instance Variables
+    @property
+    def x() -> float:
+        """Distance from origin to spot on X axis (meter)."""
+    @property
+    def y() -> float:
+        """Distance from origin to spot on Y axis (meter)."""
+    @property
+    def z() -> float:
+        """Distance from origin to spot on Z axis. (meter)"""
+    # endregion
+
+    # region Methods
+    def __init__(self, x=0.0, y=0.0, z=0.0) -> Location:
+        """Represents a spot in the world.
+
+        Args:
+            `x (float, optional)`: Distance from origin to spot on X axis (meter). Defaults to 0.0.\n
+            `y (float, optional)`: Distance from origin to spot on Y axis (meter). Defaults to 0.0.\n
+            `z (float, optional)`: Distance from origin to spot on Z axis (meter). Defaults to 0.0.\n
+
+        Returns:
+            `Location`\n
+        """
+        ...
+
+    def distance(self, location: Location) -> float:
+        """Returns Euclidean distance from this location to another one.
+
+        Args:
+            `location (Location)`: The other point to compute the distance with.\n
+
+        Returns:
+            `float`: (meters)\n
+        """
+    # endregion
+
+    # region Dunder Methods
+    def __abs__(self) -> Location:
+        """Returns a Location with the absolute value of the components x, y and z."""
+
+    def __eq__(self, __value: Location) -> bool:
+        """Returns `True` if both locations are the same point in space."""
+        ...
+
+    def __ne__(self, __value: object) -> bool:
+        """Returns `True` if both locations are different points in space."""
+        ...
+
+    def __str__(self) -> str:
+        """Parses the axis' values to string."""
+        ...
+    # endregion
+    pass
+
+
+class Map():
+    """Class containing the road information and waypoint managing. Data is retrieved from an OpenDRIVE file that describes the road. A query system is defined which works hand in hand with `carla.Waypoint` to translate geometrical information from the .xodr to natural world points. CARLA is currently working with OpenDRIVE 1.4 standard."""
+
+    # region Instance Variables
+    @property
+    def name() -> str:
+        """The name of the map. It corresponds to the .umap from Unreal Engine that is loaded from a CARLA server, which then references to the .xodr road description."""
+    # endregion
+
+    # region Methods
+    def __init__(self, name: str, xodr_content: str) -> list[Transform]:

```

Comment:
`__init__` should not return, and this is additionally wrong imo.

Fixes for all `__init__`s included in the latest pushes to my PR